packages/api-plugin-fulfillment-method-shipping-flat-rate/src/mutations/updateFlatRateFulfillmentMethod.js

```diff
@@ -0,0 +1,54 @@
+import SimpleSchema from "simpl-schema";
+import ReactionError from "@reactioncommerce/reaction-error";
+import methodSchema from "../util/methodSchema.js";
+
+const inputSchema = new SimpleSchema({
+  method: methodSchema,
+  methodId: String,
+  shopId: String
+});
+
+/**
+ * @method updateFlatRateFulfillmentMethodMutation
+ * @summary updates a flat rate fulfillment method
+ * @param {Object} context - an object containing the per-request state
+ * @param {Object} input - Input (see SimpleSchema)
+ * @returns {Promise<Object>} An object with a `method` property containing the updated method
+ */
+export default async function updateFlatRateFulfillmentMethodMutation(context, input) {
```

Comment:
Same comment about using the word Mutation in the name of the function. I understand that this may have been copy-pasted but let's try to be consistent with we do elsewhere.

Reply:
Removed